NitroxModel/Packets/InitialPlayerSync.cs

```diff
@@ -14,7 +14,7 @@ namespace NitroxModel.Packets
     public class InitialPlayerSync : Packet
     {
         public NitroxId AssignedEscapePodId { get; }
-        public List<EquippedItemData> EquippedItems { get; }
+        public Dictionary<string, NitroxId> EquippedItems { get; }
```

Comment:
`Nitrox.Test/Server/Serialization/WorldPersistenceTest.cs` is throwing errors with this change. The test needs to be addressed.

This should work:
```
            AssertHelper.IsDictionaryEqual(playerData.EquippedItems, playerDataAfter.EquippedItems, (keyValuePair, keyValuePairAfter) =>
            {
                Assert.AreEqual(keyValuePair.Key, keyValuePairAfter.Key);
                Assert.AreEqual(keyValuePair.Value, keyValuePairAfter.Value);
            });
            ```